packages/starknet-snap/src/__tests__/helper.ts

```diff
@@ -337,8 +337,12 @@ export function generateTransactionRequests({
             amount: '1',
           }),
           entrypoint: 'transfer',
+          label: 'Transfer',
+          type: 'contract',
+          chainId: '123',
```

Comment:
lets move chainId 1 level up, as there is no case for multi tx but different chain id